src/main/java/org/folio/des/converter/aqcuisition/EdifactOrdersExportConfigToTaskTriggerConverter.java

```diff
@@ -38,17 +32,20 @@ public List<ExportTaskTrigger> convert(ExportConfig exportConfig) {
     Optional.ofNullable(specificParameters.getVendorEdiOrdersExportConfig())
       .map(VendorEdiOrdersExportConfig::getEdiSchedule)
       .ifPresent(ediSchedule -> {
-        ScheduleParameters scheduleParameters = ediSchedule.getScheduleParameters();
-        if (scheduleParameters == null || !ediSchedule.getEnableScheduledExport() || NONE.equals(scheduleParameters.getSchedulePeriod())) {
-          return;
-        }
-        if (scheduleParameters.getId() == null) {
-           scheduleParameters.setId(UUID.fromString(exportConfig.getId()));
+        ScheduleParameters scheduleParams = ediSchedule.getScheduleParameters();
+        if (isScheduleTimePresent(scheduleParams)) {
+          if (scheduleParams.getId() == null) {
+            scheduleParams.setId(UUID.fromString(exportConfig.getId()));
```

Comment:
Maybe would be better to replace as:

`scheduleParams.setId(StringUtils.defaultIfBlank(scheduleParameters.getId(), UUID.fromString(exportConfig.getId())));`

Reply:
If you even think it is long, please use Objects.isNull method.

Reply:
This signature requires both strings, and we are setting UUID type, so remained all as is

Reply:
Sorry I mean ObjectUtils.defaultIfNull

Reply:
For me it's okay/ Thank you